docs/using/event-types.md

```diff
@@ -68,6 +74,9 @@ The event type has a simple JSON Schema submitted as an escaped JSON string desc
 says events will be randomly allocated to partitions, and the owner's name is
 `"acme-order-service"`. The `enrichment_strategies` array says to apply `metadata_enrichment` to submitted events (common metadata is a feature of some categories).
 
+The event type has an optional `default_statistics` object, which controls the number of partitions. Nakadi will use a sensible default if no value is provided. The values provided here 
```

Comment:
Fixed it: https://github.com/zalando/nakadi-manual/pull/46/commits/561a4c1663f8f738c5ac95014c3911bd564cf858